src/global_styling/variables/_states.scss

```diff
@@ -20,8 +20,8 @@ $ouiFocusRingSizeLarge: $ouiSizeXS !default;
 $ouiFocusRingSize: $ouiFocusRingSizeLarge * .75 !default;
 
 // Transparency
-$ouiFocusTransparency: lightOrDarkTheme(.1, .3) !default;
-$ouiFocusBackgroundColor: tintOrShade($ouiColorPrimary, ((1 - $ouiFocusTransparency) * 100%), ((1 - $ouiFocusTransparency) * 100%)) !default;
+$ouiFocusTransparency: lightOrDarkTheme(90%, 70%) !default;
```

Comment:
This naming is still misleading. How about something like `$ouiFocusChangePercent`? Because it's ultimately passed to either `shade` or `tint` functions as the `$percent` param.